### PR TITLE
Challenge details - desktop view

### PR DESCRIPTION
**Pull request by JMurphyWeb**
#720 

Gets challenge details view looking better on desktop.

### CODE REVIEW

**Comment by harrygfox** [comment]
Conflicts here @JMurphyWeb

File: public/css/main.css [hunk 1]
```diff
@@ -1404,6 +1404,10 @@ Darker-grey:    #9cafbb;
     text-align: center;
     padding-bottom: 1rem;
   }
+  .d-card--centred {
```
Comment (harrygfox) [inline public/css/main.css]: I'd prefer if this was a little bit more expressive - hard to know from the outside what a d-card is, is it like a v-card?
If it isn't expressed in the name then it should be at least described in the wiki / styleguide about the difference between a card and a d-card

File: templates/views/challenges/details.html [hunk 2]
```diff
@@ -4,48 +4,47 @@
...
+      <a class="icon icon--cancel" href="{{cancelUrl}}"></a>
+    </div>
 
-  <div class="card card--details">
-    {{#if permissions.primary}}
-      {{#if permissions.editable}}
-        <a href="/challenges/{{challenge.id}}/edit">
-          <i class="icon icon--edit icon--edit-blue icon--desktop"></i>
-        </a>
+    <div class="card card--details d-card--centred">
```
Comment (harrygfox) [inline templates/views/challenges/details.html]: Bemerror - you've got the old `d-card--centered` here but no `d-card`!

File: templates/views/challenges/details.html [hunk 3]
```diff
@@ -4,48 +4,47 @@
...
+      <h4 class="card__copy anchor anchor--shared-by anchor--right u-no-margin">Shared by
+        <a class="card__copy anchor anchor--shared-by anchor--right u-no-margin" href="/orgs/{{challenge.org_id}}">
+          {{challenge.org_name}}
+        </a>
+      </h4>
     </div>
-  {{/if}}
-    </form>
+    {{#if permissions.editable}}
+      <div class="card card--suggested-matches d-card--centred">
```
Comment (harrygfox) [inline templates/views/challenges/details.html]: Same comment here as above

File: public/css/main.css (hunk 1 above)
Comment (JMurphyWeb) [inline public/css/main.css]: yeah it is a desktop class. Can i add a comment?

Comment (JMurphyWeb) [inline public/css/main.css]: maybe i should have left it as card--centred-desktop or card--centred--desktop ???

**Pull request merged**